app/src/features/projects/view/ProjectPage.tsx

```diff
@@ -177,15 +207,48 @@ const ProjectPage: React.FC = () => {
                 </Typography>
               </Box>
             </Box>
-            <Box ml={4} mt={4} mb={4}>
-              <Button
-                variant="outlined"
-                color="primary"
-                data-testid="delete-project-button"
-                startIcon={<Icon path={mdiTrashCanOutline} size={1} />}
-                onClick={showDeleteProjectDialog}>
-                Delete Project
-              </Button>
+            <Box>
+              {projectWithDetails.project.publish_date ? (
+                <>
+                  <Box ml={4} mt={4} mb={4}>
+                    <Button
+                      variant="outlined"
+                      color="primary"
+                      data-testid="unpublish-project-button"
+                      startIcon={<Icon path={mdiToggleSwitch} size={1} />}
+                      onClick={async () => {
+                        await publishProject(false);
+                      }}>
+                      Unpublish Project
+                    </Button>
+                  </Box>
+                </>
+              ) : (
+                <>
+                  <Box ml={4} mt={4} mb={4}>
+                    <Button
+                      variant="outlined"
+                      color="primary"
+                      data-testid="publish-project-button"
+                      startIcon={<Icon path={mdiToggleSwitchOffOutline} size={1} />}
+                      onClick={async () => {
+                        publishProject(true);
+                      }}>
+                      Publish Project
+                    </Button>
+                  </Box>
```

Comment:
this can be combined into a single section with just checks to determine which text needs to be displayed and which icons need to be shown

Reply:
```
<Box ml={4} mt={4} mb={4}>
    <Button
        variant="outlined"
        color="primary"
        data-testid="publish-project-button"
        startIcon={<Icon path={projectWithDetails.project.publish_date ? mdiToggleSwitch : mdiToggleSwitchOffOutline} 
        size={1} />}
        onClick={async () => {
              if (projectWithDetails.project.publish_date) {
                      await publishProject(false);
              } else {
                      await publishProject(true);
               }
        }}>
        {projectWithDetails.project.publish_date ? 'Unpublish Project' : 'Publish Project'}
    </Button>
</Box>
```

Reply:
That worked well !  Thank you.